src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -598,6 +599,12 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
         this.selectedClinicalAttributeIds.delete(this.clinicalTrackKeyToAttributeId(clinicalTrackKey));
     }
 
+    private onTrackSortDirectionChange(trackId:TrackId, dir:number) {
+        if (dir === 1 || dir === -1) {
```

Comment:
is this right?  where is direction state stored if it's not passed to sortByData function and sortByData is called in either case?   maybe a comment would be good.

Reply:
comment added. This direction state is stored within the oncoprintjs object, this callback is called whenever the user selects a sort direction for a track. See https://github.com/cBioPortal/cbioportal/issues/3374 for more info